### ISSUE TITLE
better rename these callback types, give them some prefix like "TSimpleBle"

### ISSUE THREAD

**Issue by Alexey-T** [eriklins/Pascal-Bindings-For-SimpleBLE-Library#4]
```
type
  TCallbackScanStart = procedure(adapter: simpleble_adapter_t; userdata: PPointer);
  TCallbackScanStop = procedure(adapter: simpleble_adapter_t; userdata: PPointer);
  TCallbackScanUpdated = procedure(adapter: simpleble_adapter_t; peripheral: simpleble_adapter_t; userdata: PPointer);
  TCallbackScanFound = procedure(adapter: simpleble_adapter_t; peripheral: simpleble_adapter_t; userdata: PPointer);
```

**Comment by eriklins** [comment]
Makes sense, thanks.

**Comment by eriklins** [comment]
Fixed with commit 087cbea4e23d1d6fb673ddddf305e1a343c4ca1d
